js/generateFooter.js

```diff
@@ -0,0 +1,40 @@
+var generateFooter = {
+
+    text: {
+        1: 'Best Practices',
+        2: 'Donate Now',
+
+    },
+
+    additionalClassIfAny: {
+        1: 'bestPractices',
+        2: 'donate',
+
+    },
+
+    icon: {
+        1: 'school',
+        2: 'monetization_on',
+
+    },
+
+    link: {
+        1: '',
+        2: 'href=https://act.autismspeaks.org/site/Donation2?idb=135287805&4141.donation=form1&DONATION_LEVEL_ID_SELECTED=1&df_id=4141&mfc_pref=T&idb=[[S76:idb]]"',
+
```

Comment:
This is not an issue, more of a convenience/best-practice. You might want to use arrays instead of links for all of your dynamic content.. we can talk more via slack about this